internal/docker/docker.go

```diff
@@ -0,0 +1,101 @@
+package docker
```

Comment:
This contents of this file are mostly from the [hydroform/function](https://github.com/kyma-project/hydroform/blob/main/function/pkg/docker/run.go) package which is used in the `kyma run function` command. I could not use this package alone because a command  is executed and a directory is mounted  for each container, which is not necessary for the dashboard command. Hence, I created this file with a slightly modified functionality.

Reply:
Perfect 👍